server/methods/deleteMessage.coffee

```diff
@@ -18,7 +17,7 @@ Meteor.methods
 
 		deleteQuery = 
 			_id: message._id
-		deleteQuery['u._id'] = Meteor.userId() if user?.admin isnt true 
+		#deleteQuery['u._id'] = Meteor.userId() if user?.admin isnt true 
```

Comment:
this was an additional check, since `message` obj is passed from client, so `message?.u?._id` could be different from original message.